lumicks/pylake/point_scan.py

```diff
@@ -22,19 +23,20 @@ def _get_plot_data(self, channel):
         """Get photon count :class:`~lumicks.pylake.channel.Slice` for requested channel."""
         return getattr(self, f"{channel}_photon_count")
 
-    def _plot(self, channel, axes, **kwargs):
+    def plot(self, channel="rgb", axes=None, **kwargs):
```

Comment:
for consistency, would be nice to add `show_title` here too

Reply:
👍